config/spryk/spryks/Spryker/Glue/BackendApi/Plugin/AddGlueBackendApiResourcePlugin.yml

```diff
@@ -0,0 +1,90 @@
+spryk: template
+description: "Adds a new GlueBackendApi Resource Plugin."
+mode: both
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: Spryker
+
+    resource:
+        description: "Defines the resource name this resource plugin will work with."
+
+    className:
+        value: "{{ resource }}Resource"
+
+    extends:
+        value: \Spryker\Glue\GlueApplication\Plugin\GlueApplication\AbstractResourcePlugin
+
+    implements:
+        value: \Spryker\Glue\GlueRestApiConventionExtension\Dependency\Plugin\RestResourceInterface
```

Comment:
We need to do that differently. Since we cover both API Types (JSON and REST) we need to add a specific interface for each API Type.